docs/api.md

```diff
@@ -261,7 +261,8 @@ content-server page.
 - `redirect_uri`: Optional. If supplied, a string URL of where to redirect afterwards. Must match URL from registration.
 - `scope`: Optional. A space-separated list of scopes that the user has authorized. This could be pruned by the user at the confirmation dialog.
 - `action`: Optional. If provided, should be either `signup` or `signin`. Send to improve user experience, based on whether they clicked on a Sign In or Sign Up button.
-- `email`: Optional. If provided, this email address will be pre-filled into the account form, but the user is free to change it.
+- `email`: Optional. If provided, this email address will be pre-filled into
+  the account form, but the user is free to change it. If set to `blank`, the user must enter an email address and last signed in email address will be ignored. If not provided, the last signed in email address will be used as the default.
```

Comment:
Suggestion: "set to the literal string `blank`" may make things *slightly* clearer here.